### PR TITLE
Fix plugin activation deactivation

### PR DESCRIPTION
**Pull request by IvanGrginovInf**
Fixed acivation / deactivation classes so the plugin can be successfully installed and activated with `npx create-wp-project plugin`

### CODE REVIEW

**Review by iruzevic — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@
 /public
 /vendor
 /node_modules
-/storybook
\ No newline at end of file
+/storybook
+
+# Env specific
+.vscode/settings.json
```
Comment (dingo-d) [inline .gitignore:8]: Ideally, these kind of things (IDE settings) should be added to the global gitignore settings. We are all working on different IDEs, so it doesn't make sense to add an exclusion for every single one.

File: eightshift-boilerplate.php [hunk 2]
```diff
@@ -38,20 +41,20 @@
...
  */
 register_deactivation_hook(
-  __FILE__,
-  function() {
-    ( new \Core\Deactivate() )->deactivate();
-  }
+	__FILE__,
+	function () {
+		( new Deactivate() )->deactivate();
+	}
```
Comment (dingo-d) [inline eightshift-boilerplate.php:57]: We don't have a plugin factory for this?

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left a few questions

File: .gitignore (hunk 1 above)
Comment (IvanGrginovInf) [reply .gitignore:8]: Somebody else working on the project might not have that 🤷‍♂️  

Dunno, I'd rather if we ignored them individually.

File: eightshift-boilerplate.php (hunk 2 above)
Comment (IvanGrginovInf) [reply eightshift-boilerplate.php:57]: Nope, I could add it but I believe it's fine for now to keep things simple.

File: .gitignore (hunk 1 above)
Comment (dingo-d) [reply .gitignore:8]: Then we can have one for sublime, fop phpstorm, atom...

We should just have set up the global gitignore on our system and that's it.

File: eightshift-boilerplate.php (hunk 2 above)
Comment (dingo-d) [reply eightshift-boilerplate.php:57]: Yeah, it just looks... Messy (instantiating classes directly) 🤷🏼‍♂️

Comment (IvanGrginovInf) [reply eightshift-boilerplate.php:57]: We do the same for Main / Cli.

File: .gitignore (hunk 1 above)
Comment (IvanGrginovInf) [reply .gitignore:8]: I have some personal projects where I don't necessarily want this but ok. Added to global.

**Comment by IvanGrginovInf** [comment]
@dingo-d fixed issues.

File: .gitignore (hunk 1 above)
Comment (dingo-d) [reply .gitignore]: You can explicitly allow them in .gitignore on your personal projects 😄

File: eightshift-boilerplate.php (hunk 2 above)
Comment (dingo-d) [reply eightshift-boilerplate.php]: Yeah, I'm not too thrilled about that as well 😂

**Review by dingo-d — APPROVED** [review]

**Pull request merged**